src/Agent.Listener/Configuration/AutoLogonRegistryManager.cs

```diff
@@ -0,0 +1,339 @@
+#if OS_WINDOWS
+using Microsoft.VisualStudio.Services.Agent.Util;
+using Microsoft.Win32;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Principal;
+
+namespace Microsoft.VisualStudio.Services.Agent.Listener.Configuration
+{
+    [ServiceLocator(Default = typeof(AutoLogonRegistryManager))]
+    public interface IAutoLogonRegistryManager : IAgentService
+    {
+        void LogWarnings(string domainName, string userName);
```

Comment:
Removed this API. We are inspecting warnings while dealing with the particular setting (e.g., autologon warning while setting the autologon policies).

---
In reply to: [118589510](https://github.com/Microsoft/vsts-agent/pull/985#discussion_r118589510) [](ancestors = 118589510)